kafka-ui-react-app/src/components/common/Icons/WarningIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+
+const WarningIcon: React.FC = () => {
```

Comment:
Create new folder for `WarningIcon`
Create new file `WarningIcon.styled.ts`
Move `WarningIconContainer` to `WarningIcon.styled.ts` 
Rename `WarningIconContainer` to `Container`
Move `WarningIcon.tsx` to it's folder
Import styled as `import {* as S } from ....`

Reply:
If you want to leave this file as single file - move `WarningIconContainer` outside of `WarningIcon`
You can't have styled's inside function component's body

Reply:
https://styled-components.com/docs/basics
![image](https://user-images.githubusercontent.com/12596177/151555121-5bb868bf-cc97-45ec-883c-e5119903d001.png)

Reply:
Thx for detailed feedback! I did this.